src/map/map.h

```diff
@@ -1215,8 +1216,8 @@ END_ZEROED_BLOCK;
 	// reload config file looking only for npcs
 	void (*reloadnpc) (bool clear);
 
-	int (*check_dir) (int s_dir,int t_dir);
-	uint8 (*calc_dir) (struct block_list *src,int16 x,int16 y);
+	int (*check_dir) (enum unit_dir s_dir, enum unit_dir t_dir);
+	enum unit_dir (*calc_dir) (const struct block_list *src, int16 x, int16 y);
```

Comment:
`uint8` instead of `enum unit_dir`.

Reply:
No. We'll use enum unit_dir so it's clearer what values are allowed and to make it clearier what a direction var stands for.